doc/manuals/admin/logs.md

```diff
@@ -77,13 +77,22 @@ The different fields in each line are as follows:
     -   SUMMARY: This is a special level used by log summary traces,
         enabled with the `-logSummary` CLI option. Have a look at [the
         section on summary traces](#summary-traces) for details.
+-   **corr (correlator id)**. Can be either "N/A" (for log messages
+    "out of transaction", e.g. log lines corresponding to Orion Context
+    Broker startup), or it is a string in the UUID format.
+    An example:
+      "550e8400-e29b-41d4-a716-446655440000".
+    This 'correlator id' is either transferred from an incoming request,
+    or, if the incoming request doesn't carry any HTTP header "Fiware-Correlator",
+    the correlator is generated by the Orion context broker and then used in
+    the log file (as well as sent as HTTP header in forwarding messages and responses).
+    The correlator id is a common identifier among all services for one specific request.
 -   **trans (transaction id)**. Can be either "N/A" (for log messages
     "out of transaction", as the ones corresponding to Orion Context
-    Broker startup) or a string in the
-    format "1405598120-337-00000000001". The transaction id generation
-    logic ensures that every transaction id is unique, also for Orion
-    instances running in different VMs (which is useful in the case you
-    are aggregating logs from different sources), except if they are
+    Broker startup) or a string in the format "1405598120-337-00000000001".
+    The transaction id generation logic ensures that every transaction id is unique,
+    also for Orion instances running in different VMs (which is useful in the case
+    you are aggregating logs from different sources), except if they are
```

Comment:
And here we go again ... :-)
I don't think the description of the log file contents is the right place to describe the Fiware Correlator.
However, I will put in all these proposals in logs.md.
Perhaps some day we replace them with a link to some other part of the documentation.

BTW, the HTTP header Fiware-Correlator should also be explained somewhere, right?